### DISCUSSION TITLE
Unable to Validate Organization Domain

### DISCUSSION THREAD

**Discussion by TheMostDiligent** [community/community#23342] [New to GitHub]
<p>Hello!</p>
<p>I am having very hard times trying to validate domain for my organzation <a href="https://github.com/DiligentGraphics" rel="nofollow noopener">DiligentGraphics</a>. The instructions on the page say the following:</p>
<p>    Create a TXT record in your DNS configuration for _github-challenge-DiligentGraphics.diligentgraphics.com.</p>
<p>First of all, I am very confused about  ‘.’ at the end that is highlgithed. It appears to be part of the token, however <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/verifying-your-organizations-domain" rel="nofollow noopener">this page suggests it is not</a>.</p>
<p>I tried to create multiple variations of the record but none of them work. <a href="https://www.whatsmydns.net/#TXT/diligentgraphics.com" rel="nofollow noopener">This site shows</a> that all records have been propagated:</p>
<p>_github-challenge-DiligentGraphics.diligentgraphics.com.=9251f0d191<br>
_github-challenge-DiligentGraphics.diligentgraphics.com.=9251f0d191<br>
_github-challenge-DiligentGraphics.diligentgraphics.com=9251f0d191<br>
_github-challenge-DiligentGraphics.diligentgraphics.com=9251f0d191<br>
_github-challenge-DiligentGraphics=9251f0d191<br>
v=spf1 ip4:66.96.128.0/18 ?all</p>
<p>However, GitHub domain validation still fails</p>
<p>I tried to follow suggestions on the web, but with no luck. I tried to contact GitHub support directly, but they are not responding. I am not a system administrator and I have no idea what is wrong here and how to fix this, so any help would be appreciated.</p>

**Comment by logankilpatrick** [comment]
<p>Hey <a class="mention" href="/themostdiligent">@themostdiligent</a>, can you elaborate more on “First of all, I am very confused about  ‘.’ at the end that is highlgithed.” ? I am a little lost as to what you are referencing there. </p>

**Comment by TheMostDiligent** [comment]
<p>The conufsion comes from the fact that ‘.’ at the end of the instruction below may be both part of the token and just end of the sentence:</p>
<p>   Create a TXT record in your DNS configuration for _github-challenge-DiligentGraphics.diligentgraphics.com.</p>
<p>The syntax highlight suggests it is part of the token, however command line instruction given on <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/verifying-your-organizations-domain" rel="nofollow noopener">this page</a> in section 7 suggests there should be no dot at the end:</p>
<pre><code>$ dig _github-challenge-_ORGANIZATION_._example.com_+nostats +nocomments +nocmd TXT
</code></pre>
<p>Maybe it is the same, I don’t know. Like I said, I am not an expert here.</p>
<p>The main part of the question is that I <a href="https://www.whatsmydns.net/#TXT/diligentgraphics.com" rel="nofollow noopener">added multiple DNS record variations</a> with and without the dot, and even without <a href="http://diligentgraphics.com" rel="nofollow noopener">diligentgraphics.com</a> suffix:</p>
<p>_github-challenge-DiligentGraphics.diligentgraphics.com.=9251f0d191<br>
_github-challenge-DiligentGraphics.diligentgraphics.com=9251f0d191<br>
_github-challenge-DiligentGraphics=9251f0d191</p>
<p>But none of these variants work, and I don’t know what else to try</p>

**Comment by TheMostDiligent** [comment, marked as answer]
<p>So my problem is resolved now. Our hosting provider updated its old Web interface just in time to make DNS records editing more convenient and clear. With this new interface I was able to add the required record and verify the domain.</p>
<p>What concerns me is that since I first tried to get help from GitHub about a week ago, I haven’t got any reaction despite all my efforts to catch their attention. There is no doubts they’ve seen my post since few later posts got replies and answers, so they simply decided to ignore my question. </p>
<p>After reading through a forum a bit it turned out they sometimes don’t answer (at least in time) even more critical questions such as blocking access to a repository for no reason.</p>
<p>To me using GitHub for my business used to be a no-brainer. But after taking a more closer look at how their support treats their customers this certianly requires reconsideration.</p>
